File: MCD.java
public class MCD{
  static int mcd(int x,int y){
    if (y == 0){
      return x;
    }else{
      return mcd(y,x%y);
    }
  }
  public static void main(String[] args) {
    System.out.println(mcd(35,10));
  }
}

File: MCD.cs
using System;
namespace MCD
{
  class program
  {
    public static void Main(string[] args)
    {
        MCD(35,10);
    }
    public static void MCD(int a , int b)
    {
      int restante = 0;
      do{
        restante =b;
        b = a%b;
        a = restante;
      }while(b!=0);
      Console.WriteLine(restante);
    }
  }
}
